smart_proxy_dynflow_core.gemspec

```diff
@@ -30,7 +30,7 @@ Gem::Specification.new do |gem|
   gem.add_development_dependency('rack-test', '~> 0')
   gem.add_development_dependency('rubocop', '0.32.1')
 
-  gem.add_runtime_dependency('foreman-tasks-core', '>= 0.1')
+  gem.add_runtime_dependency('foreman-tasks-core', '>= 0.2.1')
```

Comment:
I've backported the requirements to `core-0.1.5` so that we can still release this with foreman 1.15